gers/examples/python/match_classes.py

```diff
@@ -13,17 +13,17 @@ def __init__(self, id: str, geometry:BaseGeometry, properties: dict=None) -> Non
         self.id = str(id)
         self.geometry = geometry
         self.properties = properties
-    
+
     def __str__(self) -> str:
-        return json.dumps({ 

```

Comment:
I love that your IDE is stripping trailing whitespace on save. ❤️

It's the small things in life :)